### ISSUE TITLE
GvrKeyboard text setter

### ISSUE THREAD

**Issue by vinhui** [googlevr/gvr-unity-sdk#813]
Any chance we could get a setter for `GvrKeyboard.EditorText`? I manually added it and it works, would be nice to not have to add it anytime I update.

```c#
  public string EditorText {
    get { return instance != null ? instance.keyboardState.editorText : string.Empty; }
    set { keyboardProvider.EditorText = value; }
  }
```

I need this for a custom implementation of UI textfield interaction since `InputField`s do not work.

**Comment by rusmaxham** [comment]
Looks good.  Can you make a pull request and agree to the CLA?

**Comment by vinhui** [comment]
Created pr #814

**Comment by rusmaxham** [comment]
Thank you. This change has been accepted and merged into our internal repo and will be released in a future release.

**Comment by rusmaxham** [comment]
Fixed in [v1.120](https://github.com/googlevr/gvr-unity-sdk/releases).
